### PR TITLE
CD

### PR DESCRIPTION
**Pull request by kevin1055**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+name: CI
```
Comment (blerud) [inline .github/workflows/main.yml]: this is not CI, this is CD

Comment (kevin1055) [inline .github/workflows/main.yml]:
```suggestion
name: CD
```

**Comment by kevin1055** [comment]
LGTM :shipit:

**Comment by kevin1055** [comment]
:sheep: :it:

**Comment by blerud** [comment]
🐑

**Pull request merged**